portfolio/src/main/webapp/assets/script.js

```diff
@@ -26,3 +26,20 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function collapseMenu() {
+    var i;
+    var elements = document.getElementsByClassName("collapsible");
+
+    for (i = 0; i < elements.length; i++) {
+        elements[i].addEventListener("click", function() {
```

Comment:
Stylistic choice: you can save the the for loop boilerplate and array indexing with a forEach() loop. Something like:

`elements.forEach(element => {  /** insert body of loop here **/ })`